localization/sn.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<localizationPack name="Sénégal">
+  <currencies>  
+    <currency name="West African CFA franc" iso_code="XOF" iso_code_num="4217" sign="XOF" blank="1" conversion_rate="1" format="1" decimals="0"/>
```

Comment:
Format 1 means "X0,000.00 (Such as with Dollars)"
Is this what should be applied by default? 

For the decimals, 0 means no decimals in the prices, is this ok too?